.drone.yml

```diff
@@ -0,0 +1,47 @@
+---
```

Comment:
I guess drone is what we want to use? Wonder why not GH action if we start from scratch?

Reply:
I have just picked `drone` because of familiarity. I'll have look at github actions,

Reply:
Keep in mind that we want to eventually run these components against deployed Kubernetes cluster, I think, and thus something like Circle would be probably best. However I'm fine with any of the CI to start with.

Reply:
Yes, maybe I should go with CircleCI, I want to add basic e2e tests as soon as possible.

Reply:
Cool, can K8s do it

Reply:
Unfortunately not the public offering of Drone. I don't know about github, I'll check it.

Reply:
I couldn't manage to activate github actions for our organization, since it's still an experimental feature. I'm going with CircleCI.